src/FormatNumber.elm

```diff
@@ -26,6 +48,42 @@ type alias Locale =
     }
 
 
+
+-- Locales from
```

Comment:
I didn't remove the `Locale` constructor. I simply added a few predefined ones for ease of use.

The predefined spanishLocale would actually be usefull even in the cases you describe. You would do:

```elm
lengths = { spanishLocale | decimals = 1 }
heights = { spanishLocale | decimals = 2 }
```
And it would be very readable :+1: